src/components/Button.js

```diff
@@ -1,13 +1,14 @@
-
+/* eslint-disable react/require-default-props */
 /* eslint-disable react/jsx-filename-extension */
 import PropTypes from 'prop-types';
 
 import React from 'react';
 
-export default function Button({ name }) {
-  return <button type="button">{name}</button>;
+export default function Button({ name, className }) {
+  return <button type="button" className={className}>{name}</button>;
 }
 
 Button.propTypes = {
   name: PropTypes.string.isRequired,
+  className: PropTypes.string,
 };
```

Comment:
- this component should accept `color` and `wide` as incoming prop [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/colorAndWideProps.png)]
- default `color` for button is orange. [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/colorProps.png)]. <==== this implies that you'll not apply this color via css like you did, but via `defaultProps` values (and styling inside the component), since color is passed as props
- `wide` is boolean, true means that button take 50% of space (used for `0` button) [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/WideProps.png)] <=== this means you have to enforce this boolean nature using `propTypes`. And use conditional rendering (if-else) on the button to check if `wide` is True/False. Remember, `wide` comes in as props